plugins/data_migration/frontend/public/templates/main.html

```diff
@@ -0,0 +1,30 @@
+<div class="data_migration-main">
+    <cly-header>
+        <template v-slot:header-left>
+          <h2> {{ i18n('data-migration.page-title') }}</h2>
+          &nbsp; <cly-tooltip-icon icon="ion-help-circled"></cly-tooltip-icon>
+        </template>
+        <template v-slot:header-right>
+            <el-dropdown v-if="hasCreateRight" split-button type="success" @command="handleCommand">
+                <i class="el-icon-circle-plus"></i> {{ i18n('data-migration.import-export-button-title') }}
+                <el-dropdown-menu slot="dropdown">
```

Comment:
We don't use either `<el-dropdown>` or `<el-dropdown-menu>` as they are leaking memory, please switch to `<cly-more-options>` :slightly_smiling_face: